opacus/tests/multigpu_gradcheck.py

```diff
@@ -53,20 +53,23 @@ def forward(self, x):
 def demo_basic(rank, weight, world_size, dp, clipping):
     torch.manual_seed(world_size)
     batch_size = 32
-    withdp = "with" + ("out " if not dp else "")
-    print(f"Running basic DDP {withdp} differential privacy example on rank {rank}.")
+    withdp = "with" + ("out" if not dp else "")
+    print(
+        f"Running {clipping}-clipping DDP {withdp} differential privacy example on rank {rank}."
+    )
```

Comment:
We shouldn't use prints anywhere in the library, in tests too. It may have unexpected side effects. 
If we really needs to do this output here, we should [use logging](https://docs.python.org/3/howto/logging.html):
```python
import logging
logging.info(f"Running {clipping}-clipping DDP {withdp} differential privacy example on rank {rank}.")
```